cli/cmd/lql.go

```diff
@@ -19,71 +19,217 @@
 package cmd
 
 import (
+	"fmt"
+	"io"
 	"io/ioutil"
+	"net/http"
+	"strings"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/api"
 )
 
 var (
-	lqlFile string
+	lqlEnd      string
+	lqlEnv      bool
+	lqlFile     string
+	lqlRepo     bool
+	lqlStart    string
+	lqlURL      string
+	lqlValidate bool
```

Comment:
We try to keep these flag states inside a single struct per command, just like this https://github.com/lacework/go-sdk/blob/main/cli/cmd/vulnerability.go#L35

```suggestion
	lqlCmdState = struct {
		End      string
		Env      bool
		File     string
		Repo     bool
		Start    string
		URL      string
		Validate bool
	}{}
```

Reply:
Fixed.